ui/components/app/user-preferenced-currency-display/user-preferenced-currency-display.stories.js

```diff
@@ -1,52 +1,27 @@
 import React from 'react';
-import { PRIMARY, SECONDARY, ETH } from '../../../helpers/constants/common';
+import { ETH } from '../../../helpers/constants/common';
 
 import UserPreferencedCurrencyDisplay from '.';
 
 export default {
   title: 'Components/App/UserPreferencedCurrencyDisplay',
   id: __filename,
   argTypes: {
-    className: {
-      control: 'text',
-    },
     'data-testid': {
       control: 'text',
     },
     prefix: {
       control: 'text',
     },
-    value: {
```

Comment:
There is no "container" component here, so all of these component props can be set.

I noticed that the `showFiat` prop seemingly was always missing though.